Library/Formula/webalizer.rb

```diff
@@ -1,26 +1,31 @@
-require 'formula'
-
 class Webalizer < Formula
   homepage "http://www.mrunix.net/webalizer/"
```

Comment:
Maybe better to follow the permanent redirect to this address?:
http://www.webalizer.org/